users/views.py

```
@@ -0,0 +1,62 @@
+@api_view(['POST'])
+@permission_classes([IsAuthenticated]) 
+@extend_schema(summary="로그아웃 (Refresh 토큰 필요)")
+def logout_account(request):
+    try:
+        # 클라이언트로부터 refresh 토큰을 전달받고 블렉리스트에 추가가
+        refresh_token = request.data.get("refresh")
+        token = RefreshToken(refresh_token)
+        token.blacklist()
+
+        return Response({"message": "Successfully logged out."}, status=status.HTTP_205_RESET_CONTENT)
+    except Exception as e:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)